app/res/values/attrs.xml

```diff
@@ -49,4 +49,9 @@
         <attr name="clip_height" format="reference|float"/>
     </declare-styleable>
 
+    <declare-styleable name="FilePreference">
+        <attr name="file_type" format="string"/>
+        <attr name="error_dialog_title" format="string"/>

```

Comment:
I would definitely prefer standardising it for all dialogues, though I was just trying to keep the earlier behaviour of showing dialog title specific to the feature requesting the file.